api/model.go

```diff
@@ -1951,6 +1951,15 @@ func toBackupResource(b *longhorn.Backup) *Backup {
 		return nil
 	}
 
+	backupTargetName := b.Status.BackupTargetName
+	if backupTargetName == "" {
+		backupTargetName = b.Labels[types.LonghornLabelBackupTarget]
+	}
+	volumeName := b.Status.VolumeName
+	if volumeName == "" {
+		volumeName = b.Labels[types.LonghornLabelBackupVolume]
+	}
```

Comment:
Check if `b.Labels` is empty before accessing it.

Reply:
Modified. Thanks